packages/typespec-vscode/src/const.ts

```diff
@@ -0,0 +1,9 @@
+import { SettingName } from "./types.js";
+
+export const EmitterSettingName: Record<string, string> = {
+  client: SettingName.ClientEmitter,
+  server: SettingName.ServerEmitter,
+  schema: SettingName.SchemaEmitter,
+};
+
+export const StartFileName = "main.tsp";
```

Comment:
please also help to refactor other places using this const to avoid potential future misunderstanding (people may think change this one is enough, but there are other places that actually not using it.

Reply:
updated.